test/lib/fake-collector.js

```diff
@@ -14,7 +14,7 @@ var restify = require('restify')
 var codec = require('../../lib/util/codec')
 var logger = require('../../lib/logger').child({component: 'fake_collector'})
 
-var DEFAULT_HOST = 'ssl.lvh.me'
+var DEFAULT_HOST = 'localhost'
 var ACTUAL_HOST = 'collector-1.lvh.me'
```

Comment:
does this need to get updated?

Reply:
I had that thought but seems like it is just returned from the preconnect and not really used? Certainly nothing broke and collector-1.lvh.me didn't have a cert or anything. That being said, could be updated to get rid of the .lvh to reduce confusion.

Reply:
looking at this var and the code around checking host and `ACTUAL_HOST`, this should no longer get hit and is dead code

Reply:
i'll take care of this since you had to leave unexpectedly